engine/gamesys/src/gamesys/scripts/script_resource.cpp

```diff
@@ -1917,12 +1917,17 @@ static void MakeTextureSetFromLua(lua_State* L, dmhash_t texture_path_hash, dmGr
             float geo_width = (float)CheckFieldValue<int>(L, -1, "width", 0);
             float geo_height = (float)CheckFieldValue<int>(L, -1, "height",  0);
 
-            geometry.m_PivotX = CheckFieldValue<float>(L, -1, "pivot_x", geo_width * 0.5);
-            geometry.m_PivotY = CheckFieldValue<float>(L, -1, "pivot_y", geo_height * 0.5);
+            // pivot in unit space, where (0,0) is top left of image, (1,1) is bottom right
+            float pivot_x = CheckFieldValue<float>(L, -1, "pivot_x", 0.5f);
+            float pivot_y = CheckFieldValue<float>(L, -1, "pivot_y", 0.5f);
```

Comment:
By using unit space (as in TexturePacker), we can avoid using the width/height of the image to calculate the pivot.